utils/lbryUri.js

```diff
@@ -56,15 +52,13 @@ module.exports = {
     };
   },
   parseClaim: function (claim) {
-    logger.debug('parsing name:', claim);
     const componentsRegex = new RegExp(
       '([^:$#/.]*)' + // name (stops at the first modifier)
       '([:$#.]?)([^/]*)' // modifier separator, modifier (stops at the first path separator or end)
     );
-    const [proto, claimName, modifierSeperator, modifier] = componentsRegex
+    const [, claimName, modifierSeperator, modifier] = componentsRegex
```

Comment:
Remove the extra comma

Reply:
I think it is necessary in order to ignore the first value when destructuring the returned array, right?  Otherwise claimName becomes the proto.. etc.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment